### PR TITLE
Some new (edited) trans string

### PR DESCRIPTION
**Pull request by domenc**
issue #137 (new pull request )

### CODE REVIEW

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -5,470 +5,425 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: piplmesh\n"
-"Report-Msgid-Bugs-To: https://github.com/wlanslovenija/PiplMesh\n"
-"POT-Creation-Date: 2012-06-11 00:22+0200\n"
-"PO-Revision-Date: 2012-05-18 12:29+0100\n"
-"Last-Translator: Martin Jakomin <martin.jakomin@gmail.com>\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po:8]: Don't lose our bug-reporting URL.

File: piplmesh/locale/sl/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -5,17 +5,38 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: piplmesh\n"
-"Report-Msgid-Bugs-To: https://github.com/wlanslovenija/PiplMesh\n"
-"POT-Creation-Date: 2012-05-24 15:16+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: Jon Premik <jon.premik@gmail.com>\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/djangojs.po:8]: Same.

**Comment by mitar** [comment]
Can you manually change only those translations you are really changing and not the whole file? Because now both paths and wrapping are changed everywhere in the translation files. And it is really hard to follow what you changed and what you left as it is.

**Comment by domenc** [comment]
Ok, I fix bug url, but i do not iknow why iz poedit fixing new lines string... (in new pull request)

**Comment by mitar** [comment]
Yes, but you can manually correct that. And you do not have to open a new pull request every time. Just fix in the same pull request.

**Comment by domenc** [comment]
When i push to a current branch on github i get an error so i create a new branch and PR.. (i do not know why get error now..)

**Comment by mitar** [comment]
And which error do you get?

**Comment by domenc** [comment]
git did not exit cleanly (exit code 1) ..
